chart/flux/templates/rbac.yaml

```
@@ -1,8 +1,18 @@
+{{- $kind := "Role" -}}
+{{- $bind := "RoleBinding" -}}
+{{- if .Values.clusterScope -}}
+{{ $kind = "ClusterRole" }}
```

I had to change the assignments in lines 4 and 5 to `:=` from `=` 
```suggestion
{{ $kind := "ClusterRole" }}
```

Maybe my helm client is too old?
```
Client: &version.Version{SemVer:"v2.9.1", GitCommit:"20adb27c7c5868466912eebdf6664e7390ebe710", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.13.0", GitCommit:"79d07943b03aea2b76c12644b4b54733bc5958d6", GitTreeState:"clean"}
```

---

I also had to remove the `apiGroups` from the `Role` (apparently they only work for `ClusterRoles`):

```
apiVersion: rbac.authorization.k8s.io/v1beta1
kind: {{ $kind }}
metadata:
  name: {{ template "flux.fullname" . }}
  {{- if not .Values.clusterScope }}
  namespace: {{ .Release.Namespace }}
  {{- end }}
  labels:
    app: {{ template "flux.name" . }}
    chart: {{ template "flux.chart" . }}
    release: {{ .Release.Name }}
    heritage: {{ .Release.Service }}
rules:
  - apiGroups: ["*"]
    resources: ["*"]
    verbs: ["*"]
```

---

Thank you for this effort. I'm using this PR as a base to deploy flux into a cluster with access to a limited list of namespaces. I think it might be best to create a new PR that contains my additional RBAC resources once this PR is merged.